src/main/resources/static/scripts/outputpanel.js

```diff
@@ -1,23 +1,38 @@
-function OutputPanel (classSelector, toolBarSelector, service, onFiltersChangedFromPlotFn) {
+function OutputPanel (id, classSelector, toolBarSelector, container, service, onFiltersChangedFromPlotFn) {
 
   var currentObj = this;
 
+  this.id = id;
   this.classSelector = classSelector;
   this.toolBarSelector = toolBarSelector;
   this.service = service;
   this.onFiltersChangedFromPlot = onFiltersChangedFromPlotFn;
-  this.$html = $(this.classSelector);
+  this.$html = cloneHtmlElement(id, classSelector);
+  container.html(this.$html);
+  this.$html.show();
   this.$toolBar = $(this.toolBarSelector);
   this.plots = [];
 
 
   //METHODS AND EVENTS
-  this.initPlots = function(filename, bck_filename, gti_filename, schema) {
+  this.initPlots = function(projectConfig) {
     //PLOTS HARDCODED BY THE MOMENT HERE
-    if (!filename.endsWith(".txt")) {
-      this.plots = this.getFitsTablePlots(filename, bck_filename, gti_filename, schema);
+    if (!projectConfig.filename.endsWith(".txt")) {
```

Comment:
Something strange here. We decide how to read the file based on its magic byte, so if I rename a fits file as fits.txt, in Python we will still read it as FITS, right? But here we fall back on checking the extension... Is this a bug?